IWXXM/metarSpeci.xsd

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<schema elementFormDefault="qualified" targetNamespace="http://icao.int/iwxxm/3.0" version="3.0.dev" xmlns:iwxxm="http://icao.int/iwxxm/3.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns="http://www.w3.org/2001/XMLSchema">
-	<include schemaLocation="measures.xsd"></include>
+<schema elementFormDefault="qualified" targetNamespace="http://icao.int/iwxxm/3.0" version="3.0.1RC1" xmlns:iwxxm="http://icao.int/iwxxm/3.0" xmlns:gml="http://www.opengis.net/gml/3.2" xmlns="http://www.w3.org/2001/XMLSchema">
+	<include schemaLocation="iwxxm.xsd"></include>
 	<include schemaLocation="common.xsd"></include>
```

Comment:
Is it necessary to include common.xsd again, given that it's included with iwxxm.xsd?  Seems circular.  This causes no problems?

Reply:
Good catch.  Circular reference could be an issue for some parser implementations but the case we encountered happened many years ago and I played with this arrangement for almost half a year and its seems ok.

common.xsd is included by EA automatically and if we want to get rid of it we will need to do it by post-processing script.

This is a release candidate.  May be we seek views from broader audience before deciding what to do with the imports?

Reply:
Oh~ You may want to take a look at my [last post](https://github.com/wmo-im/iwxxm/issues/213#issuecomment-666206144) in #213 in which it correctly flags the OBS time of the second volcano was behind the valid period start time but I am not sure this could happen for 2 volcano situation.

Reply:
Actually, common.xsd isn't causing the circular issue, but the inclusion of iwxxm.xsd into the product schemas, right?

Reply:
Yes.

In any case, I will remove common.xsd in the next RC if we adopted the importing of iwxxm.xsd, or revert to the original implementation.

Reply:
@moryakovdv and @jkorosi do you have any views with regard to @mgoberfield 's question?

Reply:
Seems to not bother CRUX validation tool.